src/sources/soundsourceffmpeg.cpp

```diff
@@ -152,60 +191,47 @@ SINT getStreamSeekPrerollFrameCount(const AVStream& avStream) {
     }
     case AV_CODEC_ID_AAC:
     case AV_CODEC_ID_AAC_LATM: {
-        // "AAC Audio - Encoder Delay and Synchronization: The 2112 Sample Assumption"
-        // https://developer.apple.com/library/ios/technotes/tn2258/_index.html
-        // "It must also be assumed that without an explicit value, the playback
-        // system will trim 2112 samples from the AAC decoder output when starting
-        // playback from any point in the bitsream."
-        // See also: https://developer.apple.com/library/archive/documentation/QuickTime/QTFF/QTFFAppenG/QTFFAppenG.html
-        const SINT aacSeekPrerollFrameCount = 2112;
+        const SINT aacSeekPrerollFrameCount = kavStreamDecoderFrameDelayAAC;
         return math_max(aacSeekPrerollFrameCount, defaultSeekPrerollFrameCount);
     }
     default:
         return defaultSeekPrerollFrameCount;
     }
 }
 
-QString formatErrorMessage(int errnum) {
-    char errbuf[256];
-    if (av_strerror(errnum, errbuf, sizeof(errbuf) / sizeof(errbuf[0]) == 0)) {
-        return QString("%1 (%2)").arg(errbuf, errnum);
-    } else {
-        return QString("No description for error code (%1) found").arg(errnum);
-    }
+inline std::string formatErrorString(int errnum) {
+    char errbuf[AV_ERROR_MAX_STRING_SIZE];
+    av_strerror(errnum, errbuf, sizeof(errbuf) / sizeof(errbuf[0]) == 0);
+    return std::string(errbuf);
```

Comment:
Why not user QString here. Its converted later anyway and we can save one memory allocation cycle.

Reply:
No, it is not. The plain C-string is only used passed on for logging. Why take the roundtrip to a QString/UTF-16 when all we need is a string in system encoding?